### PR TITLE
Add `<main>` HTML semantic tag term entry

### LINKED ISSUE #449: [Term Entry] HTML Tags : <main>
### Entry Type (select all that apply)

New Term for Existing Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

Adding new term entry for semantic `<main>` tag in HTML.

Closes #449

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+Title: '<main>'
+Description: 'Represents the primary content within the <body> of the web page.'
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/main/main.md:3]:
```suggestion
Description: 'Represents the primary content within the body element of the web page.'
```

Comment (yangc95) [inline content/html/concepts/semantic-html/terms/main/main.md:3]:
```suggestion
Description: 'Represents the primary content within the body element of the web page.'
```

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Below is an example of the `<main>` tag being used to display a list of notable lines from works by Shakespeare:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head> </head>
+  <body>
+    <header>
+      <img
+        src="https://cdn.pixabay.com/photo/2014/04/03/11/50/drama-312318_640.png"
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/main/main.md:28]:
```suggestion
        src="https://cdn.pixabay.com/photo/2014/04/03/11/50/drama-312318_640.png"
        width="200" height="200
```

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </li>
+        <li>
+          "Shall I compare thee to a summer’s day? Thou art more lovely and more
+          temperate." - Sonnet 1
+        </li>
+      </ul>
+    </main>
+    <footer>
+      <a href="https://en.wikipedia.org/wiki/William_Shakespeare"
+        >Learn more about William Shakespeare
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/main/main.md:47]:
```suggestion
      <a href="https://en.wikipedia.org/wiki/William_Shakespeare">
        Learn more about William Shakespeare
```

**Review by yangc95 — COMMENTED** [review]
Done with review 1️⃣ pretty solid entry so not too many comments/suggestions!

**Review by yangc95 — APPROVED** [review]
👍 Ready for review 2️⃣

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<main>` semantic tag represents the primary content within the `<body>` of the web page. This excludes content from other structural elements such as the [header](https://www.codecademy.com/resources/docs/html/semantic-html/header) or [footer](https://www.codecademy.com/resources/docs/html/semantic-html/footer).
+
```
Comment (KTom101) [inline content/html/concepts/semantic-html/terms/main/main.md:16]: To stay consistent with the other semantic HTML terms, there should be a section for "Syntax"  here before giving the example.

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    <header>
+      <img
+        src="https://cdn.pixabay.com/photo/2014/04/03/11/50/drama-312318_640.png"
+        width="200"
+        height="200"
+      />
+    </header>
+    <main>
+      <h1>My favorite Shakespeare lines</h1>
+
```
Comment (KTom101) [inline content/html/concepts/semantic-html/terms/main/main.md:35]: Is this extra space here necessary?

File: content/html/concepts/semantic-html/terms/main/main.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  <body>
+    <header>
+      <img
+        src="https://cdn.pixabay.com/photo/2014/04/03/11/50/drama-312318_640.png"
+        width="200"
+        height="200"
+      />
+    </header>
+    <main>
+      <h1>My favorite Shakespeare lines</h1>
```
Comment (KTom101) [inline content/html/concepts/semantic-html/terms/main/main.md:34]:
```suggestion
      <h1>My Favorite Shakespeare Lines</h1>
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 Great entry! The only thing that appears to be missing is a "syntax" section. In order to remain uniform with the other terms, it would be best to add one. Let me know if you have questions.

File: content/html/concepts/semantic-html/terms/main/main.md (hunk 5 above)
Comment (Dusch4593) [reply content/html/concepts/semantic-html/terms/main/main.md:35]: I usually like to keep an empty line below my `<hx>` but totally fine with removing it 😄

File: content/html/concepts/semantic-html/terms/main/main.md (hunk 4 above)
Comment (Dusch4593) [reply content/html/concepts/semantic-html/terms/main/main.md:16]: Good point, I'll add one! 😄

**Comment by Dusch4593** [comment]
@KTom101 finished w/ implementing your comments/suggestions. ✅

**Review by KTom101 — APPROVED** [review]
Thanks for the changes!

**Pull request merged**